web/src/components/table/data-table-multi-select-actions/trace-table-multi-select-action.tsx

```diff
@@ -112,6 +175,79 @@ export function TraceTableMultiSelectAction({
           </DialogFooter>
         </DialogContent>
       </Dialog>
+
+      <Dialog
+        open={addToQueueDialogOpen}
+        onOpenChange={(isOpen) => {
+          if (!isOpen) {
+            setAddToQueueDialogOpen(false);
+          }
+        }}
+      >
+        <DialogContent className="sm:max-w-md">
+          <Form {...form}>
+            <form
+              className="space-y-6"
+              onSubmit={form.handleSubmit((data) => {
+                if (data.queueId) {
+                  void mutAddToQueue
+                    .mutateAsync({
+                      projectId,
+                      queueId: data.queueId,
+                      objectIds: selectedTraceIds,
+                      objectType: "TRACE",
+                    })
+                    .then(() => {
+                      setAddToQueueDialogOpen(false);
+                    });
+                }
+              })}
+            >
+              <DialogHeader>
+                <DialogTitle>Add to Annotation Queue</DialogTitle>
+                <DialogDescription>
+                  Select an annotation queue to add the selected traces to.
+                </DialogDescription>
+              </DialogHeader>
+              <FormField
+                control={form.control}
+                name="queueId"
+                render={({ field }) => (
+                  <FormItem>
+                    <Select
+                      onValueChange={field.onChange}
+                      defaultValue={field.value}
+                    >
+                      <FormControl>
+                        <SelectTrigger>
+                          <SelectValue placeholder="Select a queue" />
+                        </SelectTrigger>
+                      </FormControl>
+                      <SelectContent>
+                        {queues?.data?.map((queue) => (
+                          <SelectItem key={queue.id} value={queue.id}>
+                            {queue.name}
+                          </SelectItem>
+                        ))}
+                      </SelectContent>
+                    </Select>
+                    <FormMessage />
+                  </FormItem>
+                )}
+              />
+              <DialogFooter className="sm:justify-start">
+                <Button
+                  type="submit"
+                  loading={mutAddToQueue.isLoading}
+                  disabled={mutAddToQueue.isLoading}
+                >
+                  Add {selectedTraceIds.length} trace(s) to queue
```

Comment:
I just had the case where I did not have a queue yet. The form showed that I need to fill out the drop down. When hitting this button, the request was still executed.

Reply:
Good catch, I'll take a look, thanks!

Reply:
I could not reproduce this behaviour, checked it a couple of times